pkg/configv2/migrate.go

```diff
@@ -0,0 +1,338 @@
+					},
+				},
+			},
+			Daemon: types.JobDefaultsConfig{
+				// TODO(forrest) [review]: Does a zero publisher value imply it has no priority or that it's not configured?
+				// Priority: 100,
+				Task: types.TaskDefaultConfig{
+					Resources: types.Resource{
+						CPU:    in.Node.Compute.Capacity.DefaultJobResourceLimits.CPU,
+						Memory: in.Node.Compute.Capacity.DefaultJobResourceLimits.Memory,
+						Disk:   in.Node.Compute.Capacity.DefaultJobResourceLimits.Disk,
+						GPU:    in.Node.Compute.Capacity.DefaultJobResourceLimits.GPU,
+					},
+					Publisher: types.DefaultPublisherConfig{
+						Type: in.Node.Requester.DefaultPublisher,
+					},
+					Timeouts: types.TaskTimeoutConfig{
+						ExecutionTimeout: types.Duration(in.Node.Compute.JobTimeouts.MaxJobExecutionTimeout),
+					},
+				},
+			},
+			Service: types.JobDefaultsConfig{
+				// TODO(forrest) [review]: Does a zero publisher value imply it has no priority or that it's not configured?
+				// Priority: 100,
+				Task: types.TaskDefaultConfig{
+					Resources: types.Resource{
+						CPU:    in.Node.Compute.Capacity.DefaultJobResourceLimits.CPU,
+						Memory: in.Node.Compute.Capacity.DefaultJobResourceLimits.Memory,
+						Disk:   in.Node.Compute.Capacity.DefaultJobResourceLimits.Disk,
+						GPU:    in.Node.Compute.Capacity.DefaultJobResourceLimits.GPU,
+					},
+					Publisher: types.DefaultPublisherConfig{
+						Type: in.Node.Requester.DefaultPublisher,
+					},
+					Timeouts: types.TaskTimeoutConfig{
+						ExecutionTimeout: types.Duration(in.Node.Compute.JobTimeouts.MaxJobExecutionTimeout),
+					},
+				},
+			},
+			Ops: types.JobDefaultsConfig{
+				// TODO(forrest) [review]: Does a zero publisher value imply it has no priority or that it's not configured?
+				// Priority: 100,
+				Task: types.TaskDefaultConfig{
+					Resources: types.Resource{
+						CPU:    in.Node.Compute.Capacity.DefaultJobResourceLimits.CPU,
+						Memory: in.Node.Compute.Capacity.DefaultJobResourceLimits.Memory,
+						Disk:   in.Node.Compute.Capacity.DefaultJobResourceLimits.Disk,
+						GPU:    in.Node.Compute.Capacity.DefaultJobResourceLimits.GPU,
+					},
+					Publisher: types.DefaultPublisherConfig{
+						Type: in.Node.Requester.DefaultPublisher,
+					},
+					Timeouts: types.TaskTimeoutConfig{
+						ExecutionTimeout: types.Duration(in.Node.Compute.JobTimeouts.MaxJobExecutionTimeout),
+					},
+				},
+			},
+		},
+		// TODO(forrest) [review]: currently both the compute and requester have a job selection policy
+		// it is not clear whose policy should be migrated here.
+		JobAdmissionControl: types.JobAdmissionControl{
+			RejectStatelessJobs: in.Node.Requester.JobSelectionPolicy.RejectStatelessJobs,
+			AcceptNetworkedJobs: in.Node.Requester.JobSelectionPolicy.AcceptNetworkedJobs,
+			ProbeHTTP:           in.Node.Requester.JobSelectionPolicy.ProbeHTTP,
+			ProbeExec:           in.Node.Requester.JobSelectionPolicy.ProbeExec,
+		},
```

Comment:
@wdbaruni how should we address this TODO?

Reply:
whichever is configured, with compute having the precedence